GLTF/src/GLTFAsset.cpp

```diff
@@ -341,6 +341,24 @@ std::vector<GLTF::BufferView*> GLTF::Asset::getAllCompressedBufferView() {
 	return compressedBufferViews;
 }
 
+void GLTF::Asset::mergeAnimations() {
+	if (animations.size() == 0) { return; }
+
+	GLTF::Animation* mergedAnimation = new GLTF::Animation();
+
+	// Merge all animations. In the future, animations should be grouped
+	// according to COLLADA <animation_clip/> nodes.
+	for (size_t i = 0; i < animations.size(); i++) {
+		GLTF::Animation* animation = animations[i];
+		for (GLTF::Animation::Channel* channel : animation->channels) {
```

Comment:
I see this pattern in this file already:

	for (GLTF::Animation* animation : animations) {
		for (GLTF::Animation::Channel* channel : animation->channels) {